src/main/java/marquez/api/ListNamespacesResponse.java

```diff
@@ -4,22 +4,28 @@
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
 import marquez.core.mappers.CoreNamespaceToApiNamespaceMapper;
-import marquez.core.models.Namespace;
 
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class ListNamespacesResponse {
-  // TODO: Remove use of CoreNamespaceToApiNamespaceMapper
-  CoreNamespaceToApiNamespaceMapper namespaceMapper = new CoreNamespaceToApiNamespaceMapper();
-  private final List<Namespace> namespaces;
 
-  // TODO: Constructor should accept marquez.api.models.Namespace instead
-  public ListNamespacesResponse(@JsonProperty("namespaces") List<Namespace> namespaces) {
-    this.namespaces = namespaces;
-  }
+  private static final CoreNamespaceToApiNamespaceMapper namespaceMapper =
+      new CoreNamespaceToApiNamespaceMapper();
+
+  @JsonProperty("namespaces")
+  @Getter(AccessLevel.NONE)
```

Comment:
Investigate why the access level is NONE. Is this necessary (here and elsewhere)?